### DISCUSSION TITLE
Which Istio configuration is checked in the attached screenshot?

### DISCUSSION THREAD

**Discussion by abonas** [kiali/kiali#3860] [Q&A]
![Screenshot at 2021-04-06 16-50-17](https://user-images.githubusercontent.com/6277245/114303079-d9a2c700-9ad4-11eb-9d3e-aef85444d83f.png)

**Comment by lucasponce** [comment]
This configuration flag shows a service validations as defined in

[1] https://kiali.io/documentation/latest/validations/#_kia0701_deployment_exposing_same_port_as_service_not_found

**Reply by abonas** [reply]
thanks. Currently it is not clear from the UI what exactly is checked.
Perhaps the tooltip message is not accurate as it says "Istio Config analyzed" which implies VS or DR, but in fact it checks other entities?

**Reply by xeviknal** [reply]
You are right. The message is not accurate for this scenario. It should be updated to fit accordingly.

**Reply by abonas** [reply]
can a discussion be converted into an issue?

**Reply by xeviknal** [reply]
It looks like you can only create a new issue and link it to this discussion. AFAIK, it can be done the other way around: issue -> discussion.

**Reply by xeviknal** [reply]
I've created the following issue: https://github.com/kiali/kiali/issues/3872
